infra/src/main/java/com/redhat/service/smartevents/infra/exceptions/BridgeErrorHelper.java

```diff
@@ -0,0 +1,53 @@
+package com.redhat.service.smartevents.infra.exceptions;
+
+import java.util.Optional;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.redhat.service.smartevents.infra.exceptions.definitions.platform.UnspecifiedProvisioningFailureException;
+
+import io.quarkus.runtime.Quarkus;
+
+@ApplicationScoped
+public class BridgeErrorHelper {
```

Comment:
I could move this into `ErrorService` or rename...

It's used to map _any_ error occurring provisioning/deprovisioning to a `BridgeError` (or use a default).